app/views/facility_statements/new.html.haml

```diff
@@ -9,7 +9,10 @@
   = render :partial => 'shared/transactions/top', :locals => { :tab => 'statements' }
 - if @order_details.any?
   = content_for :action_instructions do
-    %p.notice.notice-info= t("facility_statements.new.instructions")
+    - if SettingsHelper.feature_on?(:skip_statement_emails)
+      %p.notice.notice-info= t("facility_statements.new.instructions_without_email")
```

Comment:
Maybe use `text_helpers` and get it better organized so we can just say `text("instructions_without_email")`?